### ISSUE TITLE
FP on NO-UNDO not specified in temp-table declaration

### ISSUE THREAD

**Issue by stefandrissen** [Riverside-Software/sonar-openedge#764]
In #365 this issue was addressed and an option added to ignore a missing ``no-undo`` on temp-tables that are defined ``like`` a no-undo temp-table.

However, this rule is still flagging temp-tables that are defined like a temp-table that was defined like a no-undo temp-table.

````
DEFINE TEMP-TABLE tta NO-UNDO FIELD i AS INTEGER.
// ok
DEFINE TEMP-TABLE ttb LIKE tta.
// false positive
DEFINE TEMP-TABLE ttc LIKE ttb.
````

**Comment by gquerret** [comment]
Confirmed.
